apps/microsoft-teams/frontend/src/components/config/ContentTypeSelection/ContentTypeSelection.spec.tsx

```diff
@@ -22,7 +22,11 @@ describe('ContentTypeSelection component', () => {
   it('mounts and renders an input when a content type is selected', () => {
     const { unmount } = ContentTypeCustomRender(
       <ContentTypeSelection
-        notification={{ ...defaultNotification, contentTypeId: 'blogPost' }}
```

Comment:
not blocking, but a quick negative unit test to assert that an error message is rendered when `isItemValid()` returns false might be a small improvement.

Reply:
Great suggestion! I'll add an additional test for the `ContentTypeSelection` and `ChannelSelection` components 👍